packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -111,7 +117,14 @@ export class MessageProcessor {
       );
     }
 
-    this._graphQLCache = await getGraphQLCache(rootPath);
+    this._graphQLCache = await getGraphQLCache(rootPath, this._extensions);
+    let config = this._graphQLCache.getGraphQLConfig();
+    if (this._extensions && this._extensions.length > 0) {
+      /* eslint-disable no-await-in-loop */
+      for (const extension of this._extensions) {
+        config = await extension(config);
+      }
+    }
```

Comment:
I don't think that this is required here since we do the same thing in the `getGraphQLCache` function.

Reply:
Removed it, nice, this was a code smell with duplicate code. Moving config to cache solves this 👍